packages/fhir-router/src/fhirrouter.ts

```diff
@@ -143,31 +183,43 @@ async function patchResource(req: FhirRequest, repo: FhirRepository): Promise<Fh
   return [allOk, resource];
 }
 
+export type RestInteraction =
+  | CapabilityStatementRestInteraction['code']
```

Comment:
Low pri mental note: This is another case where having a proper `type` for a ValueSet-derived string union would have been helpful, so you could use `CapabilityStatementRestInteractionCode` rather than `CapabilityStatementRestInteraction['code']` -- I'm pretty sure the former would be faster for tsc, and probably more discoverable.